kit/signals/context.go

```diff
@@ -27,5 +26,5 @@ func WithSignals(ctx context.Context, sigs ...os.Signal) context.Context {
 
 // WithStandardSignals cancels the context on os.Interrupt, syscall.SIGTERM.
 func WithStandardSignals(ctx context.Context) context.Context {
-	return WithSignals(ctx, os.Interrupt, syscall.SIGTERM)
```

Comment:
Is there likely to be knock-on effects from this change? e.g. is this related to the signals `http/server.go` listens for? And will changing to a "hard kill" disrupt any context-saving logic we have anywhere?

Reply:
I expect no behavior change. `os.Kill == syscall.SIGTERM` on all of our platforms (surprisingly also Windows, even though the docs only say that's true for Unixes).  This is more a cleanup / a future-proofing refactor, in case we (or community members) ever want to try building for more platforms.

Reply:
I think `http/server.go` might be dead code... This is used in `cmd/influxd/launcher/cmd.go` to set the top-level context for the `influxd` process.